app/src/main/java/com/futurice/freesound/feature/search/SearchViewModel.java

```diff
@@ -124,4 +125,8 @@ private static boolean isCloseEnabled(@NonNull final String query) {
         return TextUtils.isNotNullOrEmpty(query);
     }
 
+    @NonNull
+    Observable<Option<Throwable>> getSearchErrors() {
+        return lastError.hide();
```

Comment:
Could you as well specify on what thread it should be observed on? As subscribeOn does not work properly with Subjects/Processors